include/pluto/libpluto.h

```diff
@@ -161,7 +161,8 @@ typedef struct plutoOptions PlutoOptions;
 #define MAXIMAL_FUSE 1
 /* Something in between the above two */
 #define SMART_FUSE 2
-
+/* Maximize parallelism. */
```

Comment:
"Maximize parallelism" <--  should be made more descriptive. Maximizes parallelism by grouping ...